Submitted By: Henrik S. Hansen <hsh@freecode.dk>
Date: 2004-09-02
Initial Package Version: 0.2
Upstream Status: Not submitted - simple hack
Origin: Henrik S. Hansen
Description: Adds ed to info directory.  After installing ed (make
	     install), do: (cd /usr/share/info; install-info ed.info
	     dir), since the ed makefile doesn't do install-info
	     (a bug?).
--- ed-0.2.orig/ed.texinfo	1994-11-19 13:38:01.000000000 +0100
+++ ed-0.2/ed.texinfo	2004-09-02 20:13:52.000000000 +0200
@@ -6,6 +6,11 @@
 @setchapternewpage odd
 @c %**end of header
 
+@dircategory Utilities
+@direntry
+* Ed: (ed).                     A line-oriented text editor.
+@end direntry
+
 @ifinfo
 This file documents the @code{ed} command, which has the purpose of
 editing text files.
